pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the customer portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged-in.
+* The module to be published is uploaded.
```

Comment:
s/is uploaded/has been uploaded to Pantheon/

Reply:
done